index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script  src="js/index.js"></script>
+  <br>
+  </br>
+  <main id="main">
+  <h1 id="title">
+    Dr. Norman Borlaug
+  </h1>
+  <p>The man who saved a billion lives</p>
+  <div id="img-div">
+    <img id="image" src="tribute.jpg" alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists.">
+    <figcaption id="img-caption">
+      Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+    </figcaption>
+  </div>
```

Comment:
Remember to use section tags instead of main tags.😊 This is to help you group related content. Section tags can contain any content that is repeated across documents such as sidebars, navigation links, copyright information, site logos, and search forms. This is also supported by web browsers.

HAPPY CODING.🥳